internal/util/logging/console_handler.go

```diff
@@ -180,10 +191,41 @@ func (ch *consoleHandler) WithGroup(name string) slog.Handler {
 		ga:        append(slices.Clone(ch.ga), groupOrAttrs{group: name}),
 		m:         ch.m,
 		out:       ch.out,
+		esc:       ch.esc,
 		testAttrs: ch.testAttrs,
 	}
 }
 
+// isColorized returns true if the handler is able to print colorized messages.
+func (ch *consoleHandler) isColorized() bool {
+	return ch.esc != nil
+}
```

Comment:
Why do we need that function now?

Reply:
To output non-colorized output if the `io.Writer` is not a terminal, which is a case in some tests, as well as possible in embedded FerretDB.

Reply:
I'm not asking why we need the `!=` part. But that function is used precisely once, below. That's just more code for no good reason.

Reply:
Well, readability is definitely better – there are more things to read…

Yes, we don't need a function for a single comparison in a single place, where it is clear from the context what is going on.